iampolicymanagementv1/iam_policy_management_v1_integration_test.go

```diff
@@ -1500,6 +1512,412 @@ var _ = Describe("IAM Policy Management - Integration Tests", func() {
+				ActionControlTemplateID: &testBasicActionControlTemplateID,
+			}
+
+			actionControlTemplate, response, err := service.GetActionControlTemplate(getActionControlTemplateOptions)
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(actionControlTemplate).ToNot(BeNil())
+
+			Expect(actionControlTemplate.Name).To(Equal(core.StringPtr(exampleBasicActionControlUpdateTemplateName)))
+			Expect(actionControlTemplate.AccountID).To(Equal(&testAccountID))
+			Expect(actionControlTemplate.State).To(Equal(core.StringPtr("active")))
+		})
+	})
+
+	Describe(`DeleteBasicActionControlTemplate - Delete action control template by ID`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`DeleteActionControlTemplate(DeleteActionControlTemplateOptions *DeleteActionControlTemplateOptions)`, func() {
+			deleteActionControlTemplateOptions := &iampolicymanagementv1.DeleteActionControlTemplateOptions{
+				ActionControlTemplateID: &testBasicActionControlTemplateID,
+			}
+
+			response, err := service.DeleteActionControlTemplate(deleteActionControlTemplateOptions)
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(204))
+		})
+	})
+
+	Describe(`CreateTestActionControlTemplate - Create an action control template`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`CreateTestActionControlTemplate(createActionControlTemplateOptions *CreateActionControlTemplateOptions)`, func() {
+			templateActionControl := &iampolicymanagementv1.TemplateActionControl{
+				ServiceName: core.StringPtr("am-test-service"),
+				Description: core.StringPtr("am-test-service service actionControl"),
+				Actions:     []string{"am-test-service.test.create"},
+			}
+
+			createActionControlTemplateOptions := &iampolicymanagementv1.CreateActionControlTemplateOptions{
+				Name:           &exampleActionControlTemplateName,
+				AccountID:      &testTargetEnterpriseID,
+				ActionControl:  templateActionControl,
+				Description:    core.StringPtr("Test ActionControl Template from GO SDK"),
+				Committed:      core.BoolPtr(true),
+				AcceptLanguage: core.StringPtr("default"),
+			}
+
+			actionControlTemplate, response, err := service.CreateActionControlTemplate(createActionControlTemplateOptions)
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(actionControlTemplate).ToNot(BeNil())
+			Expect(actionControlTemplate.Name).To(Equal(core.StringPtr(exampleActionControlTemplateName)))
+			Expect(actionControlTemplate.AccountID).To(Equal(core.StringPtr(testTargetEnterpriseID)))
+			Expect(actionControlTemplate.State).To(Equal(core.StringPtr("active")))
+
+			testActionControlTemplateID = *actionControlTemplate.ID
+			testActionControlTemplateVersion = *actionControlTemplate.Version
+		})
+	})
+
+	Describe(`CreateTestActionControlTemplateVersion - Create a new action control template version`, func() {
```

Comment:
Same as above, do we need an additional create action control template integration test? I think we only need one test per method (kind of the way the examples are structured).

Reply:
Added to validate schema changes in case of any API updates.